packages/dds/map/src/map.ts

```diff
@@ -362,6 +362,12 @@ export class SharedMap extends SharedObject<ISharedMapEvents> implements IShared
         this.kernel.trySubmitMessage(content, localOpMetadata);
     }
 
+    protected rebaseOpCore(content: any, localOpMetadata: unknown) {
```

Comment:
Is there a place for localMetadata?
It is not serializable in the first place, so not clear what it would mean to have it.